### PR TITLE
Use new standardize colours

### LINKED ISSUE #2636: Update existing colors to standardized colors
### Foundation Palette
By referring to Foundation Palette colors determined in the brand standards colour issue: https://github.com/MozillaFoundation/Design/issues/312#issuecomment-458746752, let's clean up and update current colors on the site.

image

You can refer to the Foundation Site color audit slide 12: https://goo.gl/jwnocf to see all the different colors that currently exist and to help locate where they appear on the site.

### Grays
On slide 13, I have made recommendations to simplify our grays, to avoid using opacity and to replace gradient with black.
image

### Colors
 On slide 13, I have made recommendations on hover shades and to simplify to a blue. 
image

**Note:** For text links, we will want to use the dark blue (for optimal color contrast and accessibility) and the hover state will be an underline not a darker shade but this will be part of the links/button standardization update.
__________________________
If you have any feedback on variable names as well, let me know. 
Additional brand color documentation on: https://foundation.mozilla.org/en/docs/brand/design/color/

### PR DESCRIPTION
**Pull request by mmmavis**
# Use New Standardize Colours 🎨 👨‍🎨 👩‍🎨 

Related to #2636 (Doesn't close it. This PR only covers parts of it)

### Design related

See [slide 14](https://docs.google.com/presentation/d/1mQJOwKj1OQkXlhHOVa2uLX0gRPQ-PffW0B3ffXztk5w/edit#slide=id.g4ed6fbb7e3_0_0) for things this PR covers

### Non-design related
- changed 'color-hex-length' linting rule to be 'long' so it's easier for devs and desiginers to compare hex codes
- placed "out of date" notice on style guide (`/style-guide`) (we've been discussing about phasing out this page so it's not worth the effort updating it anymore)

### CODE REVIEW

**Comment by mmmavis** [comment]
@Pomax @alanmoo I assigned you as code reviewers. This PR is mostly just for replacing colours so PR should be easy to review. I avoided doing code cleanup here as it will complicate things at the stage.

File: source/sass/colors.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// [TODO] Old colors to be removed soon
+
```
Comment (mmmavis) [inline source/sass/colors.scss]: This section will be removed soon. I'm leaving it untouched for now so we can easily compare what's old vs new.

Comment (Pomax) [reply source/sass/colors.scss]: Before this gets approved: is there a follow-up filed already? If not, let's file that first.

**Comment by mmmavis** [comment]
Follow-up ticket filed: https://github.com/mozilla/foundation.mozilla.org/issues/2664

**Comment by Pomax** [comment]
Just to confirm: percy shows the buyers guide colours having changed because of this, too. Is that supposed to happen @sabrinang?

**Comment by sabrinang** [comment]
@Pomax Yes we will have to update PNI colours as well to standardize the grays/colors too and I didn't know some of it would be updated already in this PR. I just filed a separate issue for updating PNI site colours here https://github.com/mozilla/foundation.mozilla.org/issues/2665

**Comment by mmmavis** [comment]
I'll update the PR to force PNI to use old blues for its links.

**Comment by mmmavis** [comment]
M E R G E D 🎉

**Pull request merged**